lib/lineup.coffee

```diff
@@ -18,6 +18,12 @@ addPage = (pageObject) ->
   keyByIndex.push key
   return key
 
+changePageIndex = (key, newIndex) ->
+  oldIndex = keyByIndex.indexOf key
+  page = keyByIndex[oldIndex]
+  keyByIndex.splice(oldIndex, 1)
+  keyByIndex.splice(newIndex, 0, page)
```

Comment:
This method looks right, but you don't need to get `page` out of the keyByIndex, since it will always be the same as `key` that is passed into this function.

Reply:
Thanks for calling this out. The change has been made.